src/frontend/src/components/RecordSearch/Record/RecordSummary/index.tsx

```diff
@@ -19,8 +24,13 @@ export default class RecordSummary extends React.Component<Props> {
     } = this.props.summary;
 
     return (
-      <div className="bg-white shadow br3 pa3 mb3">
-        <h2 className="mb3 f5 fw7">Search Summary</h2>
+      <div className="bg-white shadow br3 mb3 ph3 pb3">
+        <div className="flex justify-between">
+          <h2 className="mv3 f5 fw7">Search Summary</h2>
+            <button onClick={this.handleDownloadClick} className="ma2 ba bw1 b--light-gray mid-gray bg-white link hover-blue fw6 br3 pa1">
```

Comment:
Nice, my only suggestion is to change the padding from `pa1` to `pv1 ph2` to give some more breathing room on each end.